ApollosProject/apollos-prototype
# Fixes email field hidden by header

![2019-05-20 09 24 11](https://user-images.githubusercontent.com/1637694/58024872-3102ad00-7ae1-11e9-97a2-9cf57df7a2c8.gif)
## DESCRIPTION

### What does this PR do, or why is it needed?

We had two views trying to respond to the Keyboard, which led to an inconsistent experience. Under some situations, the email field would be hidden by the tab view header. This PR fixes that. 

### What design trade-offs/decisions were made?

n/a 
### How do I test this PR?

1. Open the app **without storybook**. 
2. Open the register or login screen.
3. Tap on a field and open the software keyboard
4. Ensure the email field is not covered.

### What automated tests did you write?

n/a
## TODO:

- [x] Closes #722 

## REVIEW:

**Manual QA**


**Code Review: Questions to consider**


> The purpose of PR Review is to _improve the quality of the software._
